File: freecodecamp/daily-code-challenge/2026-02-13.py
def get_fastest_speed(times):
    distances = [320, 280, 350, 300, 250]
    
    max_speed = 0
    segment_number = 0
    
    for i in range(5):
        speed = distances[i] / times[i]
        if speed > max_speed:
            max_speed = speed
            segment_number = i + 1
    
    return f"The luger's fastest speed was {max_speed:.2f} m/s on segment {segment_number}."

File: freecodecamp/daily-code-challenge/2026-02-13.js
function getFastestSpeed(times) {
  const distances = [320, 280, 350, 300, 250];
  
  let maxSpeed = 0;
  let segmentNumber = 0;
  
  for (let i = 0; i < 5; i++) {
    let speed = distances[i] / times[i];
    if (speed > maxSpeed) {
      maxSpeed = speed;
      segmentNumber = i + 1;
    }
  }
  
  return `The luger's fastest speed was ${maxSpeed.toFixed(2)} m/s on segment ${segmentNumber}.`;
}
